rlbot_gui/gui.py

```diff
@@ -494,6 +495,7 @@ def init_settings():
 
 def start():
     init_settings()
+    replace_upgrade_file()
```

Comment:
Sounds cool, but I have some concerns of breaking people who run the manual gui using system wide python... 
Would it not be better to move this line here?:
https://github.com/RLBot/RLBotGUI/blob/4ca195f9361d65ee7688ac5eb0f5b49b094738fa/run.py#L12

Reply:
I don't think the current code breaks anyone, but I do like your idea better, will move it.